### ISSUE TITLE
False-positive E1101 with pywintypes.error

### ISSUE THREAD

**Issue by bersbersbers** [PyCQA/pylint#2685]
### Steps to reproduce
1. pylint bug.py
[bug.py.txt](https://github.com/PyCQA/pylint/files/2744726/bug.py.txt)

```
"""Hi!"""
import pywintypes

try:
    raise pywintypes.error
except pywintypes.error:
    print('It works')
```

### Current behavior
```
************* Module bug
bug.py:5:10: E1101: Module 'pywintypes' has no 'error' member (no-member)
bug.py:6:7: E1101: Module 'pywintypes' has no 'error' member (no-member)

----------------------------------------------------------------------
Your code has been rated at -10.00/10 (previous run: -10.00/10, +0.00)
```

### Expected behavior
No error

### pylint --version output
pylint 2.2.2
astroid 2.1.0
Python 3.6.8 (tags/v3.6.8:3c6b436a57, Dec 24 2018, 00:16:47) [MSC v.1916 64 bit (AMD64)]

pywin32==224
pywin32-ctypes==0.2.0

**Comment by PCManticore** [comment]
Hi @bersbersbers Thanks for reporting an issue. Can you try running `pylint` with ``--extension-pkg-whitelist=pywintypes`` and see if that works?

**Comment by bersbersbers** [comment]
Same thing:
```
C:\Users\Downloads>pylint bug.py
************* Module bug
bug.py:5:10: E1101: Module 'pywintypes' has no 'error' member (no-member)
bug.py:6:7: E1101: Module 'pywintypes' has no 'error' member (no-member)

-------------------------------------
Your code has been rated at -10.00/10


C:\Users\Downloads>pylint --extension-pkg-whitelist=pywintypes bug.py
************* Module bug
bug.py:5:10: E1101: Module 'pywintypes' has no 'error' member (no-member)
bug.py:6:7: E1101: Module 'pywintypes' has no 'error' member (no-member)

----------------------------------------------------------------------
Your code has been rated at -10.00/10 (previous run: -10.00/10, +0.00)

```

**Comment by deajan** [comment]
I can confirm the issue, using the above test.py example:
```
"""Hi!"""
import pywintypes

try:
    raise pywintypes.error
except pywintypes.error:
    print('It works')
```

```
C:\GIT\PythonSRC>pylint test.py
************* Module test
test.py:7:0: C0304: Final newline missing (missing-final-newline)
test.py:5:10: E1101: Module 'pywintypes' has no 'error' member (no-member)
test.py:6:7: E1101: Module 'pywintypes' has no 'error' member (no-member)

-------------------------------------
Your code has been rated at -12.00/10


C:\GIT\PythonSRC>pylint --extension-pkg-whitelist=pywintypes test.py
************* Module test
test.py:7:0: C0304: Final newline missing (missing-final-newline)
test.py:5:10: E1101: Module 'pywintypes' has no 'error' member (no-member)
test.py:6:7: E1101: Module 'pywintypes' has no 'error' member (no-member)

----------------------------------------------------------------------
Your code has been rated at -12.00/10 (previous run: -12.00/10, +0.00)
```
Pylint version:
```
C:\GIT\IMF-INSTALL\PythonSRC>pylint --version
pylint 2.4.4
astroid 2.3.3
Python 3.7.6 (tags/v3.7.6:43364a7ae0, Dec 18 2019, 23:46:00) [MSC v.1916 32 bit (Intel)]
```

pywintypes version:
```
pywin32                       227
pywin32-ctypes                0.2.0
```

Anything I can help with ?
